### PR TITLE
Edaneshmand/test bench

### PR DESCRIPTION
**Pull request by Lhumd**

### CODE REVIEW

File: demos/teststand/dg_teststand_new_jump.py [hunk 1]
```diff
@@ -1,7 +1,3 @@
-## simple jump on impact test stand
```
Comment (MaximilienNaveau) [inline demos/teststand/dg_teststand_new_jump.py:1]: why did you remove these?

**Review by MaximilienNaveau — APPROVED** [review]

File: demos/teststand/dg_teststand_new_jump.py (hunk 1 above)
Comment (Lhumd) [reply demos/teststand/dg_teststand_new_jump.py:1]: This file is the same as dg_teststand_jump.py file that Avadesh implemented. I removed it in my other branch.
